### DISCUSSION TITLE
Missing  negative_prompt argument in HF demo

### DISCUSSION THREAD

**Discussion by Bigfield77** [PixArt-alpha/PixArt-alpha#40] [General]
Hello, thanks for your great work!

I just noticed that the HF demo app.py is not forwarding the negative prompt
around line 175 I added:
    negative_prompt=negative_prompt,

to my local copy and it works fine

cheers,
François

**Comment by lawrence-cj** [comment]
Thank you for your remind, we will take it back in the next commit.

**Comment by lawrence-cj** [comment]
Fix it.

**Comment by Bigfield77** [comment]
Hello,

I think it is still missing the negative_prompt argument being passed to the pipe function

images = pipe(
        prompt=prompt,
        width=width,
        height=height,
        guidance_scale=guidance_scale,
        num_inference_steps=num_inference_steps,
        generator=generator,
        num_images_per_prompt=NUM_IMAGES_PER_PROMPT,
        use_resolution_binning=use_resolution_binning,
        output_type="pil",
    ).images


thanks :)

**Reply by lawrence-cj** [reply]
Fix it
